README.md

```diff
@@ -42,23 +42,26 @@ Add the `cdk-eks-blueprint` library as as a dependency to your CDK project.
 }
 ```
 
+Run the following command to install the dependency to your local npm package manager - `npm i @shapirov/cdk-eks-blueprint`
```

Comment:
Let's match the style of the other commands and use three tics to put the command on its own line. Easier for devs to copy and paste that way ;)

Reply:
Fixed in my most recent push